locator/simple_strategy.cc

```diff
@@ -45,7 +45,11 @@ future<inet_address_vector_replica_set> simple_strategy::calculate_natural_endpo
     endpoints.reserve(replicas);
 
     for (auto& token : tm.ring_range(t)) {
-        if (endpoints.size() == replicas) {
+        // If the number of nodes in the cluster is smaller than the number of
+        // replicas, we should return the loop when endpoints already contains
```

Comment:
Hard to understand comment because when it says `replicas` I'm thinking about replica nodes, but the meaning is "number of desired replicas" (as in replication factor).